source/support.tex

```diff
@@ -1948,13 +1948,13 @@
 data race\iref{res.on.data.races}.
 \end{note}
 \begin{note}
-The order of registration could be indeterminate if \tcode{at_quick_exit} was called from more
-than one thread.
+The order of registration can be indeterminate
+if \tcode{at_quick_exit} is called from more than one thread.
 \end{note}
 \begin{note}
 The
 \tcode{at_quick_exit} registrations are distinct from the \tcode{atexit} registrations,
-and applications might need to call both registration functions with the same argument.
+and applications can call both registration functions with the same argument.
```

Comment:
This seems to change the meaning. I know it's effectively the same, but it imparts different information to the reader.

Reply:
Tried a different approach.